PharmacyManagementAPI/Repositories/PharmacyRepository.cs

```diff
@@ -1,118 +1,59 @@
-﻿using Microsoft.EntityFrameworkCore;
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
 using PharmacyManagementAPI.Entities;
 using PharmacyManagementAPI.Models;
+using AutoMapper;
 
 namespace PharmacyManagementAPI.Repositories
 {
     public class PharmacyRepository : IPharmacyRepository
     {
         private readonly PharmacyContext _context;
+        private readonly IMapper _mapper;
```

Comment:
Move the IMapper and the mapping in this file to PharmacyService. Services is the place of for all the logic and the mappings. Repositories should not have the knowledge of Models and Controller should not have knowledge of Entities.